scIB/metrics.py

```diff
@@ -1505,16 +1505,18 @@ def lisi_graph(adata, batch_key=None, label_key=None, k0=90, type_= None,
                   n_neighbors = k0, perplexity=None, subsample = subsample, 
                   multiprocessing = multiprocessing, nodes = nodes, verbose=verbose)
     
-    # iLISI: 2 good, 1 bad
-    ilisi_score = np.nanmedian(ilisi_score)
-    # cLISI: 1 good, 2 bad
-    clisi_score = np.nanmedian(clisi_score)
+    # iLISI: nbatches good, 1 bad
+    ilisi_score = np.nanmedian(lisi_score[batch_key])
```

Comment:
`lisi_score` is not defined anywhere and throwing errors. @mbuttner Did you mean `ilisi_score[batch_key]` and `clisi_score[batch_key]` respectively?